docs-src/enhancement-requests.md

```diff
@@ -0,0 +1,74 @@
+<!--
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Enhancement Tracking and Backlog
+
+Inspired by [Kubernetes enhancements](https://github.com/kubernetes/enhancements), service-api's
+provides a process for introducing new functionality or considerable changes to the project. The
+enhancement process will evolve over time as the project matures.
+
+Enhancements provides the basis of a community roadmap. Enhancements may be filed by anyone, but
+require approval from a maintainer to accept the enhancement into the project.
+
+## Quick start
+
+1. Create an [Issue](https://github.com/kubernetes-sigs/service-apis/issues/new/choose) and select
+"Enhancement Request".
+2. Follow the instructions in the enhancement request template and submit the Issue.
+
+## What is Considered an Enhancement?
+
+An enhancement is generally anything that:
+
+- impacts how a cluster is operated including addition or removal of significant
```

Comment:
probably want to change this to say 

- impacts the service apis, including addition or removal... etc

proposal lgtm, let's discuss in meeting today to give everyone a chance comment, then we can merge.